modules/ROOT/pages/manage-client-apps-latest-task.adoc

```diff
@@ -1,33 +1,27 @@
-= Managing API instance Contracts on API Manager
+
+= Approve or Reject Access Requests 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:keywords: grant access, api, contracts
 
-After you grant access to applications in Exchange, a contract is automatically created for the API. You can then manage these contracts from API Manager. +
+After the user xref:exchange::to-request-access.adoc[requests access] to your API instance from Anypoint Exchange, as the Organization Administrator, you can manually approve or reject requests from API Manager if you have configured the SLA tier for the application for manual approvals.
 
-To view your API Instance contracts:
+To approve or reject requests to access your API:
 
 . In API Manager, click *API Administration*.
+. Click the name of the API instance for which you are requesting access. 
+. Click the `<api version>:<api instance>` link, for example, `v1:16046365`. 
```

Comment:
This is not always `<api version>:<api instance>`, if the API Instance has a Label, then the label is shown.

Reply:
Done! Thanks!